packages/ui/src/components/va-button/VaButton.stories.ts

```diff
@@ -1,12 +1,367 @@
-import { defineComponent } from 'vue'
-import VaButton from './VaButton.demo.vue'
+import { VaButton } from './'
+import { userEvent } from '../../../.storybook/interaction-utils/userEvent'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
 
 export default {
```

Comment:
On `yarn test:unit` I got an error:

<img width="931" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/5142b356-1ed0-4635-b052-e5eab4cb1f59">

Seems like it doesn't fallback to default or something?